src/UIKit/UICellAccessory.cs

```diff
@@ -0,0 +1,111 @@
+//
+// UICellAccessory.cs
+//
+// Authors:
+//	Alex Soto <alexsoto@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using ObjCRuntime;
+
+#nullable enable
+#if !WATCH
+
+namespace UIKit {
+
+	public partial class UICellAccessory {
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
```

Comment:
you have a `#if !WATCH` above and `[Watch (7,0)]` attributes
only one is right :)